README.md

```diff
@@ -68,6 +68,60 @@ createdb
 ./bin/arborist --port 8080 --jwks https://dev.planx-pla.net/user/.well-known/jwks
 ```
 
+### Quickstart with Helm
```

Comment:
tbh i would rather we create a "quickstart" doc in the `docs` directory, with both the existing quickstart and the helm one, and link to it in the readme, instead of writing long readmes. Like we do in more recent services, for example https://github.com/uc-cdis/requestor/blob/master/README.md#requestor

Reply:
Hi Pauline! Apologies for missing this initially! I modified all the PRs and I added the helm quickstart guide to the docs directory and linked it from the main readme :)

Reply:
thank you 😄